docs/descriptor.rst

```diff
@@ -329,13 +329,18 @@ Getting ``tk-multi-pythonconsole`` from its ``shotgunsoftware`` Github repo:
 - ``organization`` is the Github organization or user that the repository belongs to.
 - ``repository`` is the name of the repository to find a Release for.
 - ``version`` is the name of the Release to use.
+- ``private`` is an optional setting that defines whether the repository is private and requires authentication.
 
+Private repositories are supported through the use of the Github api, authenticated with personal access tokens.
+A token must be set as environment variable that is specific to the organization setting:
+
+- ``SG_GITHUB_TOKEN_<ORGANIZATION>`` defines the personal access token used to authenticate. The organization name should be uppcase and snake cased.
```

Comment:
Just a small typo I think, `uppcase` should be `uppercase`.

Reply:
Fixed, good catch!